src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -14225,6 +14225,570 @@ public static void Main()
 5").VerifyDiagnostics();
         }
 
+        [Fact, WorkItem(65728, "https://github.com/dotnet/roslyn/issues/65728")]
+        public void DefaultParameterValue_Decimal_Lambda_DelegateIL()
```

Comment:
>DefaultParameterValue_Decimal_Lambda_DelegateIL [](http://example.com/codeflow?start=20&length=47)

Are we testing similar scenario for a method group?

Reply:
Yes, in `DefaultParameterValue_Decimal_MethodGroup`.

Reply:
> Yes, in ```DefaultParameterValue_Decimal_MethodGroup```

It looks like we are not verifying the shape of the delegate there.